spec/abstractops.html

```diff
@@ -92,6 +92,44 @@ <h1>GetStringOrNumberOption ( _options_, _property_, _stringValues_, _minimum_,
     </emu-alg>
   </emu-clause>
 
+
+  <emu-clause id="sec-temporal-durationhandlefractions" aoid="DurationHandleFractions">
+    <h1>DurationHandleFractions ( _fHours_, _minutes_, _fMinutes_, _seconds_, _fSeconds_, _milliseconds_, _fMilliseconds_, _microseconds_, _fMicroseconds_, _nanoseconds_, _fNanoseconds_ )</h1>
+    <emu-alg>
+      1. If _fHours_ is not equal to 0, then
+        1. If either of _minutes_, _fMinutes_, _seconds_, _fSeconds_, _milliseconds_, _fMilliseconds_, _microseconds_, _fMicroseconds_, _nanoseconds_, _fNanoseconds_ is not 0, throw a *RangeError* exception.
+        1. Let _mins_ be _fHours_ × 60.
+        1. Set _minutes_ to the integral Number nearest _mins_ in the direction of +0𝔽.
```

Comment:
```suggestion
        1. Set _minutes_ to the integral Number nearest to _mins_ in the direction of +0<sub>𝔽</sub>.
```
(also below)

(although I think you can also do more simply `Set _minutes_ to floor(_mins_).`)

Reply:
The linter was complaining about this line anyway so I'll push a new commit to fix it.